Feature/project mock server

euny0ung opened the pull request:
프로젝트 페이지에 mock api 적용하였습니다.

1. 기존에는 제목 아래에 부제목이 있었는데, api 항목에 부제목이 없고 저자가 있어 부제목 자리에 저자를 썼습니다.
2. 저자 자리에 '저자' 라는 글자를 넣을까 고민했는데, 검색해보니 이름을 올려도 기여도에 따라 저자가 아닌 기여자로 분류되는 
    경우가 있더라고요. 그래서 쓰지 않았습니다. 이 부분 수정이 필요한것 같다면 말해주세요.
4. 백엔드와 소통의 오류가 있었던것 같습니다. api 구조가 생각했던 것과 달라서 물어보니까 생각의 차이가 있네요...
  백엔드에서 생각한 것: 분야별로 프로젝트를 나누고 버튼을 누르면 해당되는 프로젝트 다 보여주기.    
  프론트엔드에서 생각한 것: 서울대 홈페이지처럼 진행하고 있는 프로젝트 or 대표 프로젝트 8개 보여주기. 즉, 버튼 하나에 프로 
  젝트 하나 보여주기
  그래서 코드가 조금 비효율적이긴 하지만 문제없이 구현했습니다. 이 부분은 일단 이슈에 등록해둘게요.. 


![image](https://user-images.githubusercontent.com/56199780/174488626-b8cfd1db-d6f0-4aa8-a940-48acd83ecf20.png)

euny0ung committed:
feat: Update Projects
프로젝트 페이지에 mock api를 적용하였습니다.

euny0ung committed:
fix: Update Projects
실행 됐다가 안됐다가 하는 오류 해결(useEffect 실행순서 문제)

suinj8 reviewed (commented):
API관련하여 꾸준히 백엔드와 소통해야 될거 같아요.
코드리뷰는 혼난관계로 철회했습니다..

LimEunSang commented on src/pages/research/Project.js:
```diff
@@ -107,11 +111,31 @@ function Description(props) {
 
 /**
  **@author Eunyoung-Jo, czen2@jbnu.ac.kr
```
> 고생하셨습니다. API관련하여 꾸준히 백엔드와 소통해야 될거 같아요. LGTM!

이슈 해결도 안됐는데 어떻게 LGTM 인가요
이슈에  댓글도 안 다셨던데

LimEunSang commented:
> 고생하셨습니다. API관련하여 꾸준히 백엔드와 소통해야 될거 같아요. LGTM!

이슈 해결도 안됐는데 어떻게 LGTM 인가요
이슈에 댓글도 안 다셨던데

baaaam771 commented:
프론트 디자인은 그대로 가시고
issue에 올린대로 백앤드 쪽에서 json data 형식만 바꿔주면 될것 같아요!

euny0ung committed:
fix: Fix Error
- selectComponent 부분에서 임의의 name 이름을 배열로 바꿔주었음
- 소스맵 오류로 인해 package.json 파일에서 코드 수정

#62

euny0ung committed:
fix: Update Project
- selectComponent에서 지정된 이름을 변수로 수정하였습니다.

euny0ung committed:
Merge branch 'feature/project_mock_server' of https://github.com/AI-Research-Center-Homepage/frontend into feature/project_mock_server

euny0ung committed:
feat: Update Project

euny0ung committed:
fix: Update Project
- api 데이터 가져오는 오류 수정

euny0ung committed:
fix: Fix Projects
- api 데이터가 안받아지는 오류 해결

#54

euny0ung committed:
feat: Add participants
- 프로젝트 참여자를 추가하였습니다.

#54
